instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_server.py

```diff
@@ -189,6 +188,7 @@ def _start_span(self, handler_call_details, context):
         attributes = {
             "rpc.method": handler_call_details.method,
             "rpc.system": "grpc",
+            "rpc.grpc.status_code": grpc.StatusCode.OK,
```

Comment:
Just curious as to why we set `grpc.StatusCode.OK` before the call has completed? Is the expectation that we assume it is good and if it does fail later then this status code will be replaced?

Reply:
If you don't call `abort()` or otherwise set the status code, then it's not an error, and so `OK` is the default response.  
I mean, as far as this interceptor can tell - we can't actually hook this at the true end of the call stack because interceptors don't really work that way.